pkg/operator/target_config_reconciler.go

```diff
@@ -0,0 +1,632 @@
+	required := resourceread.ReadClusterRoleBindingV1OrDie(bindata.MustAsset("assets/lws-controller-generated/rbac.authorization.k8s.io_v1_clusterrolebinding_lws-proxy-rolebinding.yaml"))
+	required.OwnerReferences = []metav1.OwnerReference{
+		ownerReference,
+	}
+
+	for i := range required.Subjects {
+		required.Subjects[i].Namespace = c.namespace
+	}
+
+	return resourceapply.ApplyClusterRoleBinding(ctx, c.kubeClient.RbacV1(), c.eventRecorder, required)
+}
+
+func (c *TargetConfigReconciler) manageIssuerCR(ctx context.Context, ownerReference metav1.OwnerReference) (*unstructured.Unstructured, bool, error) {
+	gvr := schema.GroupVersionResource{
+		Group:    "cert-manager.io",
+		Version:  "v1",
+		Resource: "issuers",
+	}
+
+	issuer, err := resourceread.ReadGenericWithUnstructured(bindata.MustAsset("assets/lws-controller-generated/cert-manager.io_v1_issuer_lws-selfsigned-issuer.yaml"))
+	if err != nil {
+		return nil, false, err
+	}
+	issuerAsUnstructured, ok := issuer.(*unstructured.Unstructured)
+	if !ok {
+		return nil, false, fmt.Errorf("issuer is not an Unstructured")
+	}
+	issuerAsUnstructured.SetNamespace(c.namespace)
+	ownerReferences := issuerAsUnstructured.GetOwnerReferences()
+	ownerReferences = append(ownerReferences, ownerReference)
+	issuerAsUnstructured.SetOwnerReferences(ownerReferences)
+
+	return resourceapply.ApplyUnstructuredResourceImproved(ctx, c.dynamicClient, c.eventRecorder, issuerAsUnstructured, c.resourceCache, gvr, nil, nil)
+}
+
+func (c *TargetConfigReconciler) manageCertificateWebhookCR(ctx context.Context, ownerReference metav1.OwnerReference) (*unstructured.Unstructured, bool, error) {
+	gvr := schema.GroupVersionResource{
+		Group:    "cert-manager.io",
+		Version:  "v1",
+		Resource: "certificates",
+	}
+
+	service := resourceread.ReadServiceV1OrDie(bindata.MustAsset("assets/lws-controller-generated/v1_service_lws-webhook-service.yaml"))
+	issuer, err := resourceread.ReadGenericWithUnstructured(bindata.MustAsset("assets/lws-controller-generated/cert-manager.io_v1_certificate_lws-serving-cert.yaml"))
+	if err != nil {
+		return nil, false, err
+	}
+	issuerAsUnstructured, ok := issuer.(*unstructured.Unstructured)
+	if !ok {
+		return nil, false, fmt.Errorf("issuer is not an Unstructured")
+	}
+	issuerAsUnstructured.SetNamespace(c.namespace)
+	ownerReferences := issuerAsUnstructured.GetOwnerReferences()
+	ownerReferences = append(ownerReferences, ownerReference)
+	issuerAsUnstructured.SetOwnerReferences(ownerReferences)
+	dnsNames, found, err := unstructured.NestedStringSlice(issuerAsUnstructured.Object, "spec", "dnsNames")
+	if !found || err != nil {
+		return nil, false, fmt.Errorf("%v: .spec.dnsNames not found: %v", issuerAsUnstructured.GetName(), err)
+	}
+	for i := range dnsNames {
+		dnsNames[i] = strings.Replace(dnsNames[i], "$(SERVICE_NAME)", service.Name, 1)
+		dnsNames[i] = strings.Replace(dnsNames[i], "$(SERVICE_NAMESPACE)", c.namespace, 1)
```

Comment:
Does this work for you? The manifests seem to be missing the `$()` part.

Reply:
also good to double check the CA injections

Reply:
Nice finding. Yes, there was a bug also in CA injections. Fixed.